### PR TITLE
Update creating-operator-manifests.md

### PR DESCRIPTION
**Pull request by fgiloux**
See [issue 159](https://github.com/operator-framework/olm-docs/issues/159). This is to have an example following best practices

### CODE REVIEW

File: content/en/docs/Tasks/creating-operator-manifests.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ spec:
     name: Example
     url: www.example.com
   version: 0.10.0
+  minKubeVersion: 1.16.0
```
Comment (timflannagan) [inline content/en/docs/Tasks/creating-operator-manifests.md:60]: I wonder if we'd want to proactively bump this minKubeVersion to something more recent so we don't have to update it again down the line.

/lgtm

Comment (fgiloux) [reply content/en/docs/Tasks/creating-operator-manifests.md]: I changed it to 1.20.0 but I don't think it matters too much as it is a minimal version

**Pull request merged**